ros_ign_gazebo_demos/CMakeLists.txt

```diff
@@ -27,4 +27,18 @@ install(
   DESTINATION share/${PROJECT_NAME}/models
 )
 
+install(
+  DIRECTORY
+    ${CMAKE_CURRENT_SOURCE_DIR}/models/vehicle
```

Comment:
How about installing the entire `models` folder? This way we don't need to update this when we add new models.